src/contracts/FieldType/DefaultDataFieldStorage.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\FieldType;
+
+use eZ\Publish\SPI\Persistence\Content\Field;
+use eZ\Publish\SPI\Persistence\Content\VersionInfo;
+
+interface DefaultDataFieldStorage
+{
+    /**
+     * Populates $field value property with default data based on the external data.
+     * $field->value is a {@link \eZ\Publish\SPI\Persistence\Content\FieldValue} object.
+     * This value holds the data as a {@link \eZ\Publish\Core\FieldType\Value} based object, according to
+     * the field type (e.g. for TextLine, it will be a {@link \eZ\Publish\Core\FieldType\TextLine\Value} object).
```

Comment:
Sadly, this doesn't render well by our chosen API reference tool - phpDocumentor, because it doesn't handle inline `@link` (or `@see` for that matter).
![image](https://github.com/ezsystems/ezplatform-kernel/assets/7099219/bdffc777-2434-4ec8-a559-906cfcddc1fa)

It renders better when changed to:
```suggestion
     * Populates $field value property with default data based on the external data.
     *
     * <code>$field->value</code> is a {@see \eZ\Publish\SPI\Persistence\Content\FieldValue} object.
     * This value holds the data as a {@see \eZ\Publish\Core\FieldType\Value} based object, according to
     * the field type (e.g. for TextLine, it will be a {@see \eZ\Publish\Core\FieldType\TextLine\Value} object).
```
1. Note that by convention the first line of method's PHPDoc is an intro and should be followed by an empty line - this is also reflected by the way phpDocumentor renders doc - the first line is actually above the method as its brief intro).
2. References to documentation elements need to be referenced via `@see`.

The end result:
![image](https://github.com/ezsystems/ezplatform-kernel/assets/7099219/aee3a608-f9ec-4af5-8012-627c30a3ba9e)